tests/E2E/test/clients/common_client.js

```diff
@@ -819,12 +819,37 @@ class CommonClient {
    * @param pause
    * @return true, if click works, false otherwise
    */
-  waitForExistAndClickJs(selector, pause = 0) {
+  waitForExistAndClickJs(selector, isXpath = true, pause = 0) {
```

Comment:
Just in order to avoid confusion, what's the difference between this function and `waitForExistAndClick()` function 🤔?

Reply:
With waitForExistAndClick, sometimes selenium click on the wrong element (example : it click on symphony dev tool and not on button Save in some forms).
The function waitForExistAndClickJs, perform a javascript click and avoid the confusion in the click (so no need to scroll before click).

Reply:
So we should almost always use `waitForExistAndClickJs` and never `waitForExistAndClick` ?

Can you write this somewhere 😉? Maybe in the function JSDoc, in order to inform other people about this fact

Reply:
done